src/main/java/bank/currency/CurrencyController.java

```diff
@@ -0,0 +1,29 @@
+package bank.currency;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Currency;
+
+@RestController
+@RequestMapping("/currencies")
+@RequiredArgsConstructor
+public class CurrencyController {
+
+    private final CurrencyConverter converter;
+
+    @GetMapping("/convert")
+    public double getCurrencyConverter(
+            @RequestParam("from") String fromCurrency,
```

Comment:
```suggestion
            @RequestParam("from") Currency fromCurrency,
```
я думаю так також запрацює